.circleci/config.yml

```diff
@@ -0,0 +1,114 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12
+    environment:
+      AUTH0_DOMAIN: zenika.eu.auth0.com
+      AUTH0_CLIENT_ID: yy1vRFMNAM2mOYb60JnQa5VB0SMW12n3
+      AUTH0_AUDIENCE: https://hasura.links.zenika.com/
+      HASURA_ENDPOINT: https://hasura.links.zenika.com
+    steps:
+      - checkout
+      - run:
+          command: npm ci
+          working_directory: web
+      - run:
+          command: npm run build
+          working_directory: web
+      - persist_to_workspace:
+          root: ./
+          paths: [web/dist/]
+      - run:
+          command: npm ci
+          working_directory: server
+      - persist_to_workspace:
+          root: ./
+          paths: [server/]
+  build-back:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+  deploy:
+    parameters:
+      clever-app-id:
+        type: string
+      hasura-clever-app-id:
+        type: string
+      deploy-dir:
+        type: string
+        default: ../deploy
+      hasura-deploy-dir:
+        type: string
+        default: ../hasura
+      hasura-endpoint:
+        type: string
+      hasura-admin-secret:
+        type: env_var_name
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: Package Hasura
+          command: |
+            mkdir --parents << parameters.hasura-deploy-dir >>
+            cp hasura/Dockerfile << parameters.hasura-deploy-dir >>/
+      - run:
+          name: Deploy Hasura
+          command: |
+            git config --global user.email "dsi@zenika.com"
+            git config --global user.name "Zenika"
+            git init
+            git add .
+            git commit -m "deploy!"
+            sudo npm install --global clever-tools
+            clever login --secret=$CLEVER_SECRET --token=$CLEVER_TOKEN
+            clever link << parameters.hasura-clever-app-id >>
+            clever deploy --force
+          working_directory: << parameters.hasura-deploy-dir >>
+      - run:
+          name: Apply migrations
+          command: |
+            npx --package hasura-cli@1.2.0-beta.3 hasura migrate apply --endpoint << parameters.hasura-endpoint >> --admin-secret ${<< parameters.hasura-admin-secret >>}
```

Comment:
The `package.json` in `./hasura` already specifies the version `hasura-cli` so you may run `npm ci` in a prior step then remove the package option here, to ensure the correct version is used.